File: 4168-mirror-distance-of-an-integer/4168-mirror-distance-of-an-integer.java
class Solution {
    public int mirrorDistance(int n) {
        return Math.abs(n - Rev(n));
    }
    private int Rev(int n) {
        int res = 0, temp = n;
        while (temp > 0) {
            res = res * 10 + temp % 10;
            temp /= 10;
        }
        return res;
    }
}

File: 4168-mirror-distance-of-an-integer/4168-mirror-distance-of-an-integer.py
class Solution:
    def get_reverse(self, n: int) -> int:
        res = 0
        while n:
            res = res * 10 + n % 10
            n = n // 10
        return res

    def mirrorDistance(self, n: int) -> int:
        return abs(n - self.get_reverse(n))
        